### ISSUE TITLE
Instrument for PhET-iO

### ISSUE THREAD

**Issue by zepumph** [phetsims/ohms-law#50]
@emily-phet and I discussed and we think that this would be a good step to take to advance the sonification side of accessibility on this sim.

https://github.com/phetsims/phet-io/blob/ee0a5b85ea06039c2871a8d3d95207c1dfc73e33/doc/how-to-instrument-a-phet-simulation-for-phet-io.md

## Overview of instrumentation steps:
 - [x] Copy this checklist to the issue description (top issue comment) for tracking. Make sure to use a permanent link like `/blob/<SHA>/` so that the specific guide you used is preserved. 
 - [x] [Understand the goal](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#understand-the-goal)
 - [x] [Code review](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#code-review)
 - [x] Use Property instead of PropertySet
 - [x] [Import Tandem to main.js](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#import-tandem-to-mainjs)
 - [x] [Create root tandem](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#create-root-tandem)
 - [x] [Pass tandems to sim screens](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#pass-tandems-to-sim-screens)
 - [x] [Pass tandems to model and view](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#pass-tandems-to-model-and-view)
 - [x] [Pass tandems to all constructors defined by the simulation](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#pass-tandems-to-all-constructors-defined-by-the-simulation)
   + See also [Testing while adding Tandem instances](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#testing-while-adding-tandem-instances). Test with `?phetioValidateTandems=false` since some common code-components don't yet have tandems.
 - [x] [Pass tandem instances to common code components](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#pass-tandem-instances-to-common-code-components)
   + See also [Testing while adding Tandem instances](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#testing-while-adding-tandem-instances).
 - [x] [Create static tandems where appropriate](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#create-static-tandems-where-appropriate).
 - [x] [Replace types with tandemized equivalents](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#replace-types-with-tandemized-equivalents)
 - [x] [Instrument any uninstrumented common-code components used by the sim](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#instrument-any-uninstrumented-common-code-components-used-by-the-sim)
 - [x] [Instrument new component types defined by the sim](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#instrument-new-component-types-defined-by-the-sim)
 - [x] [Add simulation specific support](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#add-simulation-specific-support)
 - [x] [Support dynamic state](https://github.com/phetsims/phet-io/blob/master/doc/how-to-instrument-a-phet-simulation-for-phet-io.md#support-dynamic-state)
 - [x] Make sure Joist `dt` values are used instead of Date.now() or other Date functions.  This is necessary for reproducible playback via input events.
 - [x] Are random numbers using `phet.joist.random`, and all doing so after modules are declared (non-statically)?  For instance, the following methods (and perhaps others) should not be used: `Math.random`, `_.shuffle`, `_.sample`, `_.random`
 - [x] `undefined` values cannot be saved by phet-io, sims should be written to use `null` instead
 - [x] Verify that the simulation works in all of the phet-io wrappers
 - [x] Add this sim to the list of phet-io sims in `chipper/data/test-phetio`
 - [x] Build with `grunt --brand=phet-io` and test the built version by launching build/wrappers/index and testing all the links
 - [x] Set up a task to have the newly instrumented sim tested by the QA team
 - [x] Update these instructions if you find them to be incomplete, inconsistent or incorrect
 - [ ] After publishing, add your instrumented simulation to the spreadsheet here: https://docs.google.com/spreadsheets/d/1pU9izdNQkd9vr8TvLAfXe_v68yh-7potH-y712FBPr8/edit#gid=0

**Comment by zepumph** [comment]
We just want to release this for sonification work, not necessarily testing it completely and fully. On hold until we figure out https://github.com/phetsims/phet-io/issues/1170

**Comment by zepumph** [comment]
We decided to go ahead and ready a phet-io dev version that can be used with the sonification process. I will talk with @phet-steele about what a phet-io dev test would look like. In the mean time I'll make a dev test.

**Comment by zepumph** [comment]
http://www.colorado.edu/physics/phet/dev/html/ohms-law/1.4.0-phetiodev.1/wrappers/index/ 

tested in phetsims/qa#18

**Comment by zepumph** [comment]
http://www.colorado.edu/physics/phet/dev/html/ohms-law/1.4.0-phetiodev.3/wrappers/index/

has been released. This is not a published version, but a dev version. I will mark it in the spreadsheet as fully instrumented, even though there is not yet a deliverable link on the phet-io website. Closing.
